hut_10sqft/package.xml

```diff
@@ -21,6 +21,7 @@
   <exec_depend>gnuplot-x11</exec_depend>
   <exec_depend>iftop</exec_depend>
   <exec_depend>ipython3</exec_depend>
+  <exec_depend>libwebp-dev</exec_depend>  
```

Comment:
As a heavy user of `vwebp` executable I `apt install -y libwebp-dev` manually but I don't see `vwebp` yet.

Reply:
Turned out an `apt` pkg `webp` contains `vwebp` executable. But `webp` is not available as `rosdep` key yet.

Reply:
Btw, I have a feeling Facebook may not be showing an animation out of `.webp` file even though it shows a non-animated .webp file as a static image (read this on reddit somewhere).

Reply:
Decided to add a rosdep key https://github.com/ros/rosdistro/pull/43326